src/main/java/org/apache/maven/doxia/DefaultConverter.java

```diff
@@ -426,6 +434,10 @@ private void parse(
                 throw new ConverterException("IOException: " + e.getMessage(), e);
             }
         }
+        LOGGER.info(
```

Comment:
ditto

Reply:
For me this is crucial information when a directory has been given as input. For a single file it should not matter (as it is only one log line) and also clarifies which absolute file has been used (as input parameter is often relative).